src/components/interaction-designer/resource-editors/ResourceEditor.vue

```diff
@@ -1,68 +1,84 @@
 
         <template v-for="contentType in discoverContentTypesFor(mode)">
           <!-- todo: it's odd that we pass around a ContentType variant rather than a ContentTypeLangMode variant (aka, mode as external arg) -->
 
-          <resource-variant-text-editor :resource-id="resource.uuid"
-                                        :resource-variant="findOrGenerateStubbedVariantOn(
-                                                resource,
-                                                {language_id: languageId, content_type: contentType, modes: [mode]})"
+          <resource-variant-text-editor
+            v-if="contentType === SupportedContentType.TEXT"
+            :resource-id="resource.uuid"
 
-                                        :mode="mode"
-                                        v-if="contentType === SupportedContentType.TEXT"
-                                        :enable-autogen-button="true || enableAutogenButton" />
+            :resource-variant="findOrGenerateStubbedVariantOn(
+              resource,
+              {language_id: languageId, content_type: contentType, modes: [mode]})"
+            :mode="mode"
+            :enable-autogen-button="true || enableAutogenButton" />
 
           <div v-if="contentType === SupportedContentType.AUDIO">
             <template v-if="!findAudioResourceVariantFor(resource, {language_id: languageId, content_type: contentType, modes: [mode]})">
-              <upload-monitor :uploadKey="`${block.uuid}:${languageId}`" />
+              <upload-monitor :upload-key="`${block.uuid}:${languageId}`" />
 
               <ul class="nav nav-tabs">
                 <li class="nav-item">
-                  <a class="nav-link px-2 py-1 active" @click.prevent="" href="#">{{'flow-builder.library' | trans}}</a>
+                  <a
+                    class="nav-link px-2 py-1 active"
+                    href="#"
+                    @click.prevent="">{{ 'flow-builder.library' | trans }}</a>
                 </li>
-                <li v-if="can(['edit-content', 'send-call-to-records'], true) && isFeatureAudioUploadEnabled" class="nav-item">
-                  <a @click.prevent="triggerRecordViaPhoneFor(languageId)" href="#" class="nav-link px-2 py-1">{{'flow-builder.phone-recording' | trans}}</a>
+                <li
+                  v-if="can(['edit-content', 'send-call-to-records'], true) && isFeatureAudioUploadEnabled"
+                  class="nav-item">
+                  <a
+                    href="#"
+                    class="nav-link px-2 py-1"
+                    @click.prevent="triggerRecordViaPhoneFor(languageId)">{{ 'flow-builder.phone-recording' | trans }}</a>
                 </li>
 
-                <li class="nav-item" v-if="isEditable">
-                  <a v-if="isFeatureAudioUploadEnabled"
-                     class="nav-link px-2 py-1"
-                     v-flow-uploader="{
+                <li
+                  v-if="isEditable"
```

Comment:
Right now, the rule is if you have 1 attribute, it can be on 1 line. any more than that, and line breaks are needed. When in doubt, run code cleanup command in PHPStorm, then do ESLint autofix, and it will put it in the correct format. Technically all modified files should now have code cleanup command in PHPStorm, then ESLint autofix on all changed files before committing changes (this can be automated to happen on save of files in PHPStorm). This way, the code format is always correct, and we never add additional tech debt due to mixed code styles.